hack/templates/dev-builds.yaml

```diff
@@ -110,7 +110,7 @@ objects:
       dockerStrategy:
         from:
           kind: ImageStreamTag
-          name: centos:centos7
+          name: centos:7
```

Comment:
Probably, it was an annoyance for me when I was testing this which is why it ended up committed.  I think if we're going to update all the sources for dev-builds it should be a separate PR.  Until I updated these `centos7` -> `7` they wouldn't even build.